cas-server-core/src/main/java/org/jasig/cas/util/UniqueTicketIdGenerator.java

```diff
@@ -22,14 +22,13 @@
  * Interface that enables for pluggable unique ticket Ids strategies.
  *
  * @author Scott Battaglia
-
  * @since 3.0.0
- * <p>
- * This is a published and supported CAS Server 3 API.
- * </p>
  */
 public interface UniqueTicketIdGenerator {
-
+    /**
+     * Default instance for the ticket id generator.
+     */
+    UniqueTicketIdGenerator DEFAULT = new DefaultUniqueTicketIdGenerator();
```

Comment:
not sure why you would attach this to the interface as its all about the specific implementation having a fallback default not there being a default ticket generator.